SparkPost/2web2ui
# AC-1935 - Organize tracking logic and enhance submit event for testing

### What Changed

- Moved the new Form component into a subfolder called tracking. Any future components related to tracking activity should be put here too. (Higher order tracking components that wrap the main components - to keep all tracking logic in one central location)
- Logging the form information but only for non production ENVs if the local debug is set.

### How To Test

- Check that the Form component is still imported correctly.
- Check that the FieldSet component is still imported correctly.
- Check that the logging in the Form submit event is only happening on non production envs (for now.. that part is temporary until we actually hook up the tracking call and pass info into it per Form instance.)

### To Do
